torchao/dtypes/utils.py

```diff
@@ -143,3 +143,15 @@ def _get_layout_tensor_constructor(cls: Callable, layout_type_class: type(Layout
 
 def is_device(target_device_str: str, device: Union[str, torch.device]):
     return torch.device(device).type == target_device_str
+
+def get_out_shape(input_shape: Tuple[int], weight_shape: Tuple[int]) -> Tuple[int, int]:
```

Comment:
why is this related to weight_shape? I thought this is just something like `tmp = x_vals_int8.reshape(-1, x_vals_int8.shape[-1])`